### PR TITLE
Add dhbw-mannheim.de

### PR DESCRIPTION
**Pull request by DavidLiedke**

### CODE REVIEW

**Comment by DavidLiedke** [comment]
> ERROR src/chrome/content/rules/dhbw-mannheim.de.xml: Fetch error: http://alumni.dhbw-mannheim.de/ => https://alumni.dhbw-mannheim.de/: (60, 'SSL certificate problem: unable to get local issuer certificate')

I dont know why Travis get this error. The certificate looks good to me.

**Comment by DavidLiedke** [comment]
@J0WI do you have an idea?

**Comment by J0WI** [comment]
The server sends a wrong certificate chain. This does not affect browsers, where the correct intermediate cert is already cached, but `firefox --new-instance --profile $(mktemp -d) https://alumni.dhbw-mannheim.de/`, `curl https://alumni.dhbw-mannheim.de/` or [SSL Labs](https://www.ssllabs.com/ssltest/analyze.html?d=alumni.dhbw-mannheim.de) will fail.

File: src/chrome/content/rules/dhbw-mannheim.de.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		www.wf.dhbw-mannheim.de
+		www.wi.dhbw-mannheim.de
+		www.wiw.dhbw-mannheim.de
+		www.zeeb.dhbw-mannheim.de
+		www.zemath.dhbw-mannheim.de
+
+-->
+<ruleset name="DHBW Mannheim (partial)">
+
+	<!-- dhbw-mannheim.de -->
```
Comment (J0WI) [inline src/chrome/content/rules/dhbw-mannheim.de.xml]: Please also add `dhbw-mannheim.de` (without `www`) as target.

**Comment by DavidLiedke** [comment]
@J0WI dhbw-mannheim.de added as target.

**Pull request merged**